gaiku-3d/src/bakers/voxel/mod.rs

```diff
@@ -210,27 +194,57 @@ impl Baker for VoxelBaker {
     }
 }
 
+/// Either get the vertex at this position or inser one.
```

Comment:
```suggestion
/// Either get the vertex at this position or insert one.
```

Reply:
Yes I have a few typos there's even one in the title of the amethyst demo app... :(